Lib/ctypes/_aix.py

```diff
@@ -0,0 +1,265 @@
+"""Lib/ctypes support for LoadLibrary interface to dlopen() for AIX
+Similar kind of support (i.e., as a separate file)
+as has been done for Darwin support ctypes.macholib.*
+rather than as separate, detailed if: sections in utils.py
+
+dlopen() is an interface to AIX initAndLoad() - primary documentation at:
+https://www.ibm.com/support/knowledgecenter/en/ssw_aix_61/com.ibm.aix.basetrf1/dlopen.htm
+https://www.ibm.com/support/knowledgecenter/en/ssw_aix_61/com.ibm.aix.basetrf1/load.htm
+"""
+__author__ = "Michael Felt <aixtools@felt.demon.nl>"
+__version__ = "1.0.0"
+
+# Latest Update (comments): 13 October 2016
+# Thanks to Martin Panter for his patience and comments
+
+from re import search, match, escape
+from os import environ, path
+from sys import executable
+from ctypes import c_void_p, sizeof
+from ctypes._util import _last_version
+from subprocess import Popen, PIPE
```

Comment:
OK. Long time ago - but I thought I was following some guidelines (maybe not PEP guidelines).

To be clear - replace ONLY search with re.search, or do you mean all related calls such as:
executable -> sys.executable, c_void_p and sizeof to ctypes.c_void_p and ctypes.sizeof

And regarding _last_version - ctypes._util._last_version (this routine was suggested by Martin, rather than a copy of the routine that it came from (and hence removed from util.py)

iirc from what I read before - the reason to not import the whole module had to do with memory consumption and/or performance (i.e., importing the whole module is (relatively) expensive.

So, awaiting your reply.

Reply:
Please replace all imports.

> iirc from what I read before - the reason to not import the whole module had to do with memory consumption and/or performance (i.e., importing the whole module is (relatively) expensive.

I never heard anything like this.